python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2545,3 +2545,21 @@ def test_parquet_columns_and_index_param(index, columns):
     got = cudf.read_parquet(buffer, columns=columns)
 
     assert_eq(expected, got, check_index_type=True)
+
+
+def test_parquet_writer_zstd():
+    size = 12345
+    expected = cudf.DataFrame(
+        {
+            "a": np.arange(0, stop=size, dtype="int64"),
+            "b": np.random.choice(list("abcd"), size=size),
+            "c": np.random.choice(np.arange(4), size=size),
+        }
+    )
+    try:
+        buff = BytesIO()
+        expected.to_orc(buff, compression="ZSTD")
+        got = cudf.read_orc(buff)
+        assert_eq(expected, got)
+    except RuntimeError:
```

Comment:
Nit: Would be nice to scope the `try/except RuntimeError` to the line(s) that could raise the exception (same for the test above)